cumulus/parachains/runtimes/collectives/collectives-westend/src/lib.rs

```diff
@@ -741,7 +835,8 @@ pub type TxExtension = (
 	frame_system::CheckEra<Runtime>,
 	frame_system::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
-	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
+	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+	frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
```

Comment:
Is this needed? I understand it is better to have it. But I am afraid it breaks stuff.

Reply:
Off-chain workers don't need this?